packages/app-builder/src/routes/ressources+/scenarios+/$scenarioId+/testrun+/create.tsx

```diff
@@ -58,18 +61,49 @@ export async function action({ request, params }: ActionFunctionArgs) {
       }),
     );
   } catch (error) {
-    return json(submission.reply());
+    if (isStatusConflictHttpError(error)) {
```

Comment:
TBH I did this first, and then I thought to just disable the button. Perhaps we can leave this here anyway, not sure what's best.

Reply:
Use ternary